test/test.spec.js

```diff
@@ -1,6 +1,6 @@
 import * as chai from 'chai';
-import sinon from 'sinon';
-import sinonChai from 'sinon-chai';
+import * as sinon from 'sinon';
```

Comment:
Using `import *` for these `sinon` modules was actually a mistake, rolled it back later.